integration_tests/e2e/login.cy.ts

```diff
@@ -65,6 +65,6 @@ context('SignIn', () => {
     cy.task('stubAuthUser', 'bobby brown')
     cy.signIn()
 
-    indexPage.headerUserName().contains('B. Brown')
+    indexPage.headerUserName().contains('J. Smith')
```

Comment:
Hmmm, we might want to look at this .... how does this work? If we sign in as Bobby Brown (line 65), how/why does the header say "J. Smith" .... hmmmm ....
I'll checkout your branch and take a look ....

Reply:
Ah, I see, its the wiremock stub .... we might be able to do something there , let me take a look ....